lib/skate/detours/db/detour.ex

```diff
@@ -75,22 +117,31 @@ defmodule Skate.Detours.Db.Detour do
 
     def select_detour_list_info(query \\ base()) do
       query
-      |> select_merge([
+      |> preload([
+        :author
+      ])
```

Comment:
note: I _feel_ like there should be a better way to tell if we need to preload the author, (and preload using a join instead) but I wasn't able to figure it out in time.

Reply:
[`ab0e852` (#2948)](https://github.com/mbta/skate/pull/2948/commits/ab0e852919f6ccc2b8b71699dab884df324e2c75)